src/scripts/main.js

```diff
@@ -1,3 +1,69 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('.field');
+const addRow = document.querySelector('.append-row');
+const remRow = document.querySelector('.remove-row');
+const addColumn = document.querySelector('.append-column');
+const remColumn = document.querySelector('.remove-column');
+const firstRow = table.rows[0];
```

Comment:
Using `firstRow` directly assumes that the table always has at least one row. Consider checking if the table has rows before accessing `table.rows[0]` to avoid potential errors if the table is empty.